ocs_ci/deployment/deployment.py

```diff
@@ -183,16 +186,43 @@ def post_ocp_deploy(self):
         add_stage_cert()
         if config.ENV_DATA.get("huge_pages"):
             enable_huge_pages()
+        # setting up dummy zone labels
+        if config.DEPLOYMENT.get("dummy_zone_node_labels"):
+            logger.info("trying to setup dummy_zone_node_labels")
+            if are_zone_labels_missing():
+                to_label = [
+                    ("master_availability_zones", get_master_nodes()),
+                    ("worker_availability_zones", get_worker_nodes()),
+                ]
+                for zone_opt, nodes in to_label:
+                    zones = config.ENV_DATA.get(zone_opt)
+                    if zones is None:
+                        msg = f"{zone_opt) is not defined in ENV_DATA conf"
+                        logger.error(msg)
+                        raise UnexpectedDeploymentConfiguration(msg)
+                    assign_dummy_zones(zones, nodes)
+            else:
+                # don't use dummy zone labeling on a cluster with actuall zones
+                msg = (
+                    "Cluster in unexpected state before dummy zone labeling: "
+                    "at least one node already have a zone label."
+                )
+                logger.error(msg)
+                raise UnexpectedDeploymentConfiguration(msg)
+
```

Comment:
Can you please move this part to the function and from here just call this function? In that function's docstring you can document also what it exactly raises in which cases.

You can maybe have whole this function in the new module you defined: ocs_ci/deployment/zones.py WDYT?

Reply:
Ok.